lib/methods/import-page.js

```diff
@@ -143,7 +143,10 @@ const update = async ({
         ...move
       }
     }),
-    _id
+    _id,
+    {
+      fetchRelationships: false
```

Comment:
This call is using a method that was never intended for programmatic use, so that call should have been refactored in core when import-export was written. But at this point the quick fix is to introduce a way to pass this missing option programmatically. We got away without it until now because of a lack of enforcement of certain schema requirements in core.